src/compoz.ts

```diff
@@ -317,13 +317,16 @@ export class Compoz {
 			}
 		}
 
-		this.elInput.onkeyup = () => {
+		this.elInput.onkeyup = e => {
 			const contentHTML = this.getContentHTML()
 			if (contentHTML === "") {
 				this.disableButtonSend()
 			} else {
 				this.enableButtonSend()
 			}
+			if (e.keyCode === 90) {
```

Comment:
What is 90?

Reply:
Its the keyCode for (ctrl + Z)

Reply:
We should add it as comment or as constant, instead of number.